dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -256,6 +256,16 @@ void RepoqueryCommand::set_argument_parser() {
         "Limit to unneeded installed packages (i.e. packages that were installed as "
         "dependencies but are no longer needed).",
         false);
+    extras = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this, "extras", '\0', "Limit to packages that are not present in any of of available repositories.", false);
```

Comment:
Maybe we should be more explicit here and state that the packages are **installed** - `Limit to installed packages...`. Does it make the description too long?
And there is a duplicated **of** - `...in any of of available...`

Reply:
I like it, I have also rephrased the ending a bit.